static/src/js/components/PublishPreview/PublishPreview.jsx

```diff
@@ -339,6 +383,47 @@ const PublishPreview = () => {
               ]
             }
           />
+          <CustomModal
+            show={showTagModal}
+            toggleModal={toggleTagModal}
+            actions={
+              [
+                {
+                  label: 'Close',
+                  variant: 'primary',
+                  onClick: () => { toggleTagModal(false) }
+                }
+              ]
+            }
+            header={previewMetadata.tagDefinitions?.items && `${Object.keys(previewMetadata.tagDefinitions.items).length} ${pluralize('tag', Object.keys(previewMetadata.tagDefinitions.items).length)}`}
+            message={
+              previewMetadata.tagDefinitions
+                ? (
+                  <>
+                    <h3 className="fw-bolder h5">{}</h3>
+                    <ListGroup>
+                      <For each={previewMetadata.tagDefinitions.items}>
+                        {
+                          (tagItems) => (
+                            <ListGroupItem key={tagItems.tagKey}>
```

Comment:
Double check this. I'm getting duplicate keys from this one and it looks like it's using a token?
![Screenshot 2024-03-06 at 1 48 02 PM](https://github.com/nasa/mmt/assets/135638667/c17582e6-48c3-4732-87bb-c89deaa87e12)

Reply:
Interesting too since I only have one tag...

Reply:
I looked into it and found this is coming from edsc-metadata-preview. I checked in MMT and we are not using the token as a key